azerothcore/azerothcore-wotlk
# fix(DB/Waypoints): Rare Shadowforge Commander was not patrolling

I added some routes to the 4 spawns he have.



## Changes Proposed:
-  Added some patrolling to his spawn points
-  Changed his movement speed so he dont speedwalk in the base

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7031

## SOURCE:


## Tests Performed:

- I checked his patrols to see if he was walking in a good way and didnt collide into walls

Small video test

https://user-images.githubusercontent.com/87535580/126638698-8a71a4db-7bb2-4618-b32e-15c1d226cc53.mp4

## How to Test the Changes:


1. .go c id 2744
( If you cant find him try: .go c 69114,   .go c 134453 .go c 134454 or .go c 134455)
2. Check if the routes are good, so he dont go into objects or moves weird


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
